Userland/Applications/Browser/History.cpp

```diff
@@ -13,25 +13,28 @@ void History::dump() const
     dbgln("Dump {} items(s)", m_items.size());
     int i = 0;
     for (auto& item : m_items) {
-        dbgln("[{}] {} {}", i, item, m_current == i ? '*' : ' ');
+        dbgln("[{}] {} '{}' {}", i, item.url, item.title, m_current == i ? '*' : ' ');
         ++i;
     }
 }
 
-void History::push(const URL& url)
+void History::push(const URL& url, String title)
```

Comment:
`title` should be a const reference or `move()`d below - whichever you do, please do the same with `url` for consistency.